lib/containers/docker.pm

```diff
@@ -56,8 +46,8 @@ sub build_img() {
 sub test_built_img() {
     assert_script_run("mkdir /root/templates");
     assert_script_run "curl -f -v " . data_url('containers/index.html') . " > /root/templates/index.html";
-    assert_script_run("docker run -dit -p 8888:5000 -v /root/templates:\/usr/src/app/templates myapp https://openqa.suse.de//api/v1/jobs/${id}");
+    assert_script_run("docker run -dit -p 8888:5000 -v ~/templates:\/usr/src/app/templates myapp www.google.com");
```

Comment:
Cool, this looks like a more robust solution.
I must say I liked the openQA api approach more but it seems that at some point, cloning the test to a local machine would give a headache inducing failure to someone.

Reply:
Yes, previous version was giving problems in local envs. This works as long as there is internet connectivity, which is for most cases.